### ISSUE TITLE
MFTF test cases have parsing error

### ISSUE THREAD

**Issue by developer-vtnetzwelt** [magento/magento2#20067]
### Preconditions (*)

I have set up the MFTF on my localhost according to magento devdocs.
1. Magento version 2.3
https://devdocs.magento.com/mftf/2.3/getting-started.html

### Steps to reproduce (*)

1. Follow the steps mentioned in the devdocs to setup MFTF.
2. In the end, at Run test cases command i am getting parser error 
 Couldn't parse test '/var/www/html/magento2mftf/dev/tests/acceptance/tests/functional/Magento/FunctionalTest/_generated/default/EndToEndB2CGuestUserTestCest.php' on line 869  
  syntax error, unexpected '{'                                                                                                                                                   
                                

### Expected result (*)

1.  Test cases should be run without any error.
2. These are complete magento code, there is no customisation done till now.

### Actual result (*)

1.  Couldn't parse test '/var/www/html/magento2mftf/dev/tests/acceptance/tests/functional/Magento/FunctionalTest/_generated/default/EndToEndB2CGuestUserTestCest.php' on line 869  
  syntax error, unexpected '{'                                                                                                                                                                         
2. I need to rename this file so that i can run other test cases easily.

**Comment by magento-engcom-team** [comment]
Hi @developer-vtnetzwelt. Thank you for your report. 
To help us process this issue please make sure that you provided the following information:
- [ ] Summary of the issue
- [ ] Information on your environment
- [ ] Steps to reproduce
- [ ] Expected and actual results

Please make sure that the issue is reproducible on the vanilla Magento instance following **Steps to reproduce**. To deploy vanilla Magento instance on our environment, please, add a comment to the issue: 

`@magento-engcom-team give me $VERSION instance` 

where `$VERSION` is [version tags](https://github.com/magento/magento2/tags) (starting from 2.2.0+) or develop branches (for example: 2.3-develop). 
For more details, please, review the [Magento Contributor Assistant](https://devdocs.magento.com/guides/v2.2/contributor-guide/contributing.html#contributor-assist) documentation.

@developer-vtnetzwelt do you confirm that you was able to reproduce the issue on vanilla Magento instance following steps to reproduce?
- [ ] yes
- [ ] no

**Comment by magento-engcom-team** [comment]
Hi @engcom-backlog-nazar. Thank you for working on this issue. 
In order to make sure that issue has enough information and ready for development, please read and check the following instruction: :point_down:
- [x] 1. Verify that issue has all the required information. (**Preconditions**, **Steps to reproduce**, **Expected result**, **Actual result**).<details><summary>Details</summary>If the issue has a valid description, the label `Issue: Format is valid` will be added to the issue automatically. Please, edit issue description if needed, until label `Issue: Format is valid` appears.</details>

- [x] 2. Verify that issue has a meaningful description and provides enough information to reproduce the issue. If the report is valid, add `Issue: Clear Description` label to the issue by yourself.

- [x] 3. Add `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.
- [ ] 4. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [x] 5. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [x] 6. Add label `Issue: Confirmed` once verification is complete. 

- [x] 7. Make sure that automatic system confirms that report has been added to the backlog.

**Comment by magento-engcom-team** [comment]
@engcom-backlog-nazar Thank you for verifying the issue. Based on the provided information internal tickets `MAGETWO-97434` were created

**Comment by magento-engcom-team** [comment]
Hi @irajneeshgupta. Thank you for working on this issue. 
Looks like this issue is already verified and confirmed. But if your want to validate it one more time, please, go though the following instruction:
- [ ] 1. Add/Edit `Component: XXXXX` label(s) to the ticket, indicating the components it may be related to.

- [ ] 2. Verify that the issue is reproducible on `2.3-develop` branch<details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.3-develop instance` to deploy test instance on Magento infrastructure. <br/> - If the issue is reproducible on `2.3-develop` branch, please, add the label `Reproduced on 2.3.x`.<br/>- If the issue is not reproducible, add your comment that issue is not reproducible and close the issue and **_stop verification process here_**!
</details>

- [ ] 3. Verify that the issue is reproducible on `2.2-develop` branch. <details><summary>Details</summary>- Add the comment `@magento-engcom-team give me 2.2-develop instance` to deploy test instance on Magento infrastructure. <br/>- If the issue is reproducible on `2.2-develop` branch, please add the label `Reproduced on 2.2.x`</details>

- [ ] 4. If the issue is not relevant or is not reproducible any more, feel free to close it.

**Comment by sdzhepa** [comment]
As a possible fix could be next:

Where: https://github.com/magento/magento2/blob/2.3-develop/app/code/Magento/Checkout/Test/Mftf/ActionGroup/CheckoutActionGroup.xml#L189

Add: 
```
<arguments>
    <argument name="subtotal" type="string"/>
    <argument name="shippingTotal" type="string"/>
    <argument name="shippingMethod" type="string"/>
    <argument name="total" type="string"/>
</arguments>
```

Instead:
```
<arguments>
    <argument name="subtotal"/>
    <argument name="shippingTotal"/>
    <argument name="shippingMethod"/>
    <argument name="total"/>
</arguments>
```

**Comment by hws47a** [comment]
Parsing error is already fixed in commit dd7c19ee5bfb218c5de32072f490df781dc00db7 and not reproducible anymore

**Comment by sivaschenko** [comment]
Closed as not actual anymore
